app/templates/gulpfile.js

```diff
@@ -112,6 +122,22 @@ gulp.task('serve:dist', function () {
   });
 });
 
+gulp.task('serve:tests', function(){
+  browserSync({
+    notify: false,
+    port: 9090,
```

Comment:
Lemme check the docs and test this, but I believe that Browser Sync grabs `port` _and_ `port + 1` by default (the second port is for the new management interface). If my suspicion is correct, I'll happily submit a patch to `generator-webapp` to maintain consistency.